styles.css

```diff
@@ -0,0 +1,90 @@
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+
+}
+.app {
+    width: 100vw;
+    height: 100vh;
+    background: url(images/background.png);
+    background-repeat: no-repeat;
+    background-size: cover;
+}
+
+.nav-bar-list {
+    display: flex;
+    gap: 32px;
+   align-items: center;
+}
+.nav-bar-list-right {
+    display: flex;
+    gap: 12px;
+   align-items: center;
+}
+.nav-bar {
+    max-width: 1440px;
+    height: 72px;
+    display: flex;
+    align-items: center;
+    padding: 0 20px;
+    justify-content: space-between;
+    margin: auto;
+}
+.nav-link {
+    list-style: none;
+}
+
+.nav-link-text {
+    font-family: 'Inter', sans-serif;
+    font-weight: 500;
+    font-size: 14px;
+    line-height: 20px;
+    color: #18191F;
+    text-decoration: none;
+}
+.hero-section {
+    display: flex;
+    flex-direction: column;
+    max-width: 1440px;
+    margin: 100px auto auto;
+    padding: 0 20px;
+
+}
+.hero-title {
+    font-family: 'Inter', sans-serif;
+    font-weight: 800;
+    font-size: 72px;
+    line-height: 98px;
+    color: #18191F;
+    margin-top: 70px;
+}
+.hero-content {
+    gap: 32px;
+    display: flex;
+    flex-direction: column;
+}
+.hero-button {
+    width: 154px;
+    height: 48px;
+    background: #8C30F5;
+    border-radius: 6px;
+    color: #FFFFFF;
+    border: 0;
+    font-family: 'Inter', sans-serif;
+    font-weight: 500;
+    font-size: 16px;
+    line-height: 24px;
+    cursor: pointer;
+}
+.hero-content-wrapper {
+    display: flex;
+    justify-content: space-between;
```

Comment:
same here, better to use justifty-content:space-around;

Reply:
it's the same idea here.